libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayEventSimulator.kt

```diff
@@ -17,10 +19,14 @@ import kotlinx.coroutines.launch
 /**
  * This class keeps track of a forward playing replay. As time moves forward, it captures
  * all events from [ReplayEvents] that happened, and provides them in a [ReplayEvents]
+ *
```

Comment:
Could we add KDoc for `replayEvents`? I know it's `internal` but for consistency it'd be great to have 😉